src/templates/en/index.html

```diff
@@ -191,6 +231,25 @@ <h1>Methodology</h1>
         </div>
       </section>
       <hr>
```

Comment:
Two changes to the `hr`s in the footer:

- lower the opacity to 20%
- should have the same horizontal padding as other footer sections, not extending to the edges of the viewport